stratum/hal/lib/tdi/tofino/tofino_chassis_manager.cc

```diff
@@ -94,82 +96,95 @@ ::util::Status TofinoChassisManager::AddPortHelper(
+                                              singleton_port.speed_bps(),
+											  config_params.fec_mode()));
+  LOG(INFO) << "Added port " << port_id << " in node " << node_id
             << " (SDK Port " << sdk_port_id << ").";
-  RETURN_IF_ERROR(tdi_sde_interface_->AddPort(
-      unit, sdk_port_id, singleton_port.speed_bps(), config_params.fec_mode()));
+
   config->speed_bps = singleton_port.speed_bps();
   config->admin_state = ADMIN_STATE_DISABLED;
   config->fec_mode = config_params.fec_mode();
 
   if (config_params.mtu() != 0) {
-    RETURN_IF_ERROR(
-        tdi_sde_interface_->SetPortMtu(unit, sdk_port_id, config_params.mtu()));
+    RETURN_IF_ERROR(tdi_sde_interface_->SetPortMtu(device, sdk_port_id,
+	              	                               config_params.mtu()));
+    LOG(INFO) << "Set MTU " << config_params.mtu() << " for port " << port_id
+            << " in node " << node_id << " (SDK Port " << sdk_port_id << ").";
   }
   config->mtu = config_params.mtu();
   if (config_params.autoneg() != TRI_STATE_UNKNOWN) {
     RETURN_IF_ERROR(tdi_sde_interface_->SetPortAutonegPolicy(
-        unit, sdk_port_id, config_params.autoneg()));
+        device, sdk_port_id, config_params.autoneg()));
+    LOG(INFO) << "Set autoneg policy " << TriState_Name(config_params.autoneg())
+            << " for port " << port_id << " in node " << node_id
+            << " (SDK Port " << sdk_port_id << ").";
   }
   config->autoneg = config_params.autoneg();
 
   if (config_params.loopback_mode() != LOOPBACK_STATE_UNKNOWN) {
-    LOG(INFO) << "Setting port " << port_id << " to loopback mode "
-              << config_params.loopback_mode() << " (SDK Port " << sdk_port_id
-              << ").";
     RETURN_IF_ERROR(tdi_sde_interface_->SetPortLoopbackMode(
-        unit, sdk_port_id, config_params.loopback_mode()));
+        device, sdk_port_id, config_params.loopback_mode()));
+    VLOG(1) << "Set loopback mode "
+            << LoopbackState_Name(config_params.loopback_mode()) << " for port "
+            << port_id << " in node " << node_id << " (SDK Port " << sdk_port_id
+            << ").";
+
+
   }
   config->loopback_mode = config_params.loopback_mode();
 
   if (config_params.admin_state() == ADMIN_STATE_ENABLED) {
-    LOG(INFO) << "Enabling port " << port_id << " in node " << node_id
-              << " (SDK Port " << sdk_port_id << ").";
-    RETURN_IF_ERROR(tdi_sde_interface_->EnablePort(unit, sdk_port_id));
+
+    RETURN_IF_ERROR(tdi_sde_interface_->EnablePort(device, sdk_port_id));
     config->admin_state = ADMIN_STATE_ENABLED;
+    LOG(INFO) << "Enabled port " << port_id << " in node " << node_id
+              << " (SDK Port " << sdk_port_id << ").";
   }
 
   RETURN_IF_ERROR(
-      tdi_sde_interface_->EnablePortShaping(unit, sdk_port_id, TRI_STATE_FALSE));
+      tdi_sde_interface_->EnablePortShaping(device, sdk_port_id,
+	                                        TRI_STATE_FALSE));
```

Comment:
Broken indentation (one of many)

Reply:
fixed the indentation and you are right it was a tab in a file